components/src/core/components/CardTable/Cell/__tests__/__snapshots__/checkbox.spec.ts.snap

```diff
@@ -2,6 +2,6 @@
 
 exports[`CardTable > Cell > Checkbox.vue should renders OXD CardTable > Cell > Checkbox Cell 1`] = `
 <div class="oxd-checkbox-wrapper"><label class="">
-    <!--v-if--><input type="checkbox" value="0"><span class="oxd-checkbox-input oxd-checkbox-input--active --label-right oxd-checkbox-input"><i class="oxd-icon oxd-icon--medium bi-check oxd-checkbox-input-icon"></i></span>
```

Comment:
This icon change is fine, since discussed and agreed with mike

Reply:
resolving this as discussed and agreed with Mike